tests/unit/bokeh/test___init__.py

```diff
@@ -78,7 +80,7 @@ def test___version___type() -> None:
 def test___version___defined() -> None:
     assert b.__version__ != 'unknown'
 
-def test_license(capsys) -> None:
+def test_license(capsys: Capture) -> None:
```

Comment:
What benefit is provided by typing pytest fixtures?

Reply:
The benefit can be seen two lines later:
```py
out, err = capsys.readouterr()
assert out == _LICENSE
```
Now you know what `capsys` is and what's its API, and you don't have to run this code or search for "pytest capsys" to know. In fact, I hate this implicit fixture API, as it's often unclear what the thing is and where it came from (is it pytest's or thirpardy). I'm pretty sure some IDEs are clever and can special-case such features, but I prefer this to be self-evident.